### PR TITLE
Added white color to Getting started link

### PR DESCRIPTION
**Pull request by montogeek**
Implements #30 

I couldn't find or detect a CSS styleguide, so I just add a new class.
I removed link previous class because they were dead code.

### CODE REVIEW

**Comment by montogeek** [comment]
Thanks!

File: content/index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ toc: false
 	<p>Fast 3kb alternative to React with the same ES6 API.</p>
 
 	<p>
-		<a href="/guide/getting-started" class="btn btn-primary btn-lg">Getting Started</a>
+		<a href="/guide/getting-started" class="white">Getting Started</a>
 		&nbsp; • &nbsp;
```
Comment (montogeek) [inline content/index.md]: @developit Something happened and now these spaces are missing in the webpage. Git diff is clean, weird.

It use to look like this:
https://cloud.githubusercontent.com/assets/1002461/18014137/2fdef6ca-6b87-11e6-953e-c949d92112dd.png

Now spaces around the dot are missing:
screen shot 2016-09-19 at 09 17 26

Comment (developit) [inline content/index.md]: Seems like it was an update to `preact-markup`. Fixing!

Comment (montogeek) [inline content/index.md]: Cool, I thought I was my fault :D

Comment (developit) [inline content/index.md]: nope :)

**Pull request merged**